requirements.txt

```diff
@@ -2,6 +2,7 @@ Django==1.11.23
 django-bootstrap3==11.0.0
 git+https://github.com/inducer/django-bootstrap3-datetimepicker.git
 django-crispy-forms==1.7.2
+django-debug-toolbar==1.11
```

Comment:
How did this get in here?

Reply:
First time I started Osler in a Virtual Environment it threw an error that said missing django-debug-toolbar. I've added it in requirements.txt, probably I've didn't use the developer mode.